### PR TITLE
fix: add timing traces

### PR DESCRIPTION
**Pull request by chris13524**
# Description

- Implement timing traces to see what's slowing down requests.
- Refactor to avoid redundant `ResolveIdentityParams` struct.
- Add `fmt-imports` target to keep code formatting

## How Has This Been Tested?

Not tested

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/handlers/identity/resolve.rs [hunk 1]
```diff
@@ -13,19 +13,15 @@ use {
...
 pub struct ResolveIdentityPayload {
     #[serde(rename = "publicKey")]
-    public_key: String,
-}
-
-#[derive(Validate, Debug)]
-pub struct ResolveIdentityParams {
     #[validate(custom = "validate_identity_key")]
-    identity_key: String,
+    public_key: String,
```
Comment (Elyniss) [inline src/handlers/identity/resolve.rs:24]: why?

Comment (Elyniss) [reply src/handlers/identity/resolve.rs:24]: You think that has a time impact?

Comment (chris13524) [reply src/handlers/identity/resolve.rs:24]: Cus don't need redundant structs

Comment (chris13524) [reply src/handlers/identity/resolve.rs:24]: No I don't think it impacts the time significantly

File: justfile [hunk 2]
```diff
@@ -42,6 +42,17 @@ fmt:
   @echo '==> Reformatting code'
   cargo +nightly fmt
 
+fmt-imports:
```
Comment (Elyniss) [inline justfile:45]: should we add it to lint?

**Review by Elyniss — APPROVED** [review]

**Pull request merged**